pkg/clustermanager/cluster_manager.go

```diff
@@ -1121,6 +1125,20 @@ func (c *ClusterManager) ApplyBundles(ctx context.Context, clusterSpec *cluster.
 	return nil
 }
 
+// ApplyReleases applies the EKSARelease manifest.
+func (c *ClusterManager) ApplyReleases(ctx context.Context, clusterSpec *cluster.Spec, cluster *types.Cluster) error {
```

Comment:
does this need to be exported?

Reply:
yes, it's needed in the upgrade workflow